app/controllers/issues_controller.rb

```diff
@@ -183,7 +195,7 @@ def set_or_initialize_tags
   end
 
   def issue_params
-    params.require(:issue).permit(:state, :tag_list, :text)
+    params.require(:issue).permit(:state, :tag_list, :text, :redirect_to)
```

Comment:
No need for this if we're not nesting the `redirect_to` param under `:issue`

```suggestion
    params.require(:issue).permit(:state, :tag_list, :text)
```

Reply:
fixed